src/gc/gc.cpp

```diff
@@ -1973,7 +1973,7 @@ BOOL Aligned (size_t n)
 #define OBJECT_ALIGNMENT_OFFSET (sizeof(MethodTable *))
 
 #ifdef FEATURE_STRUCTALIGN
-#define MAX_STRUCTALIGN OS_PAGE_SIZE
+#define MAX_STRUCTALIGN GC_PAGE_SIZE
```

Comment:
Why should it be OS page size?

It sounds odd that the max alignment supported by the GC would vary based on the OS kernel you are running on. I would think that this should be GC_PAGE_SIZE. We do not ever built with `FEATURE_STRUCTALIGN` today so it does not matter much.